index.bs

```diff
@@ -47,84 +97,72 @@ interface Socket {
+    const socket = connect("google.com:80");
 
-      const reader = socket.readable.getReader();
+    const reader = socket.readable.getReader();
 
-      while (true) {
-        const { value, done } = await reader.read();
-        if (done) {
-          // the ReadableStream has been closed or cancelled
-          break;
-        }
-        // In many protocols the \`value\` needs to be decoded to be used:
-        const decoder = new TextDecoder();
-        console.log(decoder.decode(value));
+    while (true) {
+      const { value, done } = await reader.read();
+      if (done) {
+        // the ReadableStream has been closed or cancelled
+        break;
       }
+      // In many protocols the \`value\` needs to be decoded to be used:
+      const decoder = new TextDecoder();
+      console.log(decoder.decode(value));
+    }
 
-      reader.releaseLock();
-    ```
-  </aside>
+    reader.releaseLock();
+  </pre>
+</div>
 
 The ReadableStream operates in non-byte mode, that is the `type` parameter to the
 ReadableStream constructor is not set.
 This means the stream's controller is {{ReadableStreamDefaultController}}.
 
-`writable` attribute
---------------------
+<h4 id="writable-attribute">writable</h4>
 
-  <aside class="example">
-    The below example shows typical {{WritableStream}} usage to write data to a socket:
+The {{writable}} attribute is a {{WritableStream}} which sends data to the server the socket is connected to.
 
-    ```javascript
-      import { connect } from 'sockets';
-      const socket = connect("google.com:80");
+<div class="example">
+  The below example shows typical {{WritableStream}} usage to write data to a socket:
 
-      const writer = socket.writable.getWriter();
-      const encoder = new TextEncoder();
-      writer.write(encoder.encode("GET / HTTP/1.0\r\n\r\n"));
-    ```
-  </aside>
+  <pre highlight="js">
+    import { connect } from 'sockets';
+    const socket = connect("google.com:80");
+
+    const writer = socket.writable.getWriter();
+    const encoder = new TextEncoder();
+    writer.write(encoder.encode("GET / HTTP/1.0\r\n\r\n"));
+  </pre>
+</div>
 
-`closed` attribute
---------------------
+<h4 id="closed-attribute">closed</h4>
 
-The `closed` promise can be used to keep track of the socket state. It gets resolved under the
+The {{closed}} attribute is a promise can be used to keep track of the socket state. It gets resolved under the
 following circumstances:
 
-* the `close` method is called on the socket
-* the socket was constructed with the `allowHalfOpen` parameter set to `false`, the ReadableStream
-    is being read from, and the remote connection sends a FIN packet (graceful closure) or a RST
-    packet
+<ul>
+  <li>the {{close()}} method is called on the socket</li>
+  <li>the socket was constructed with the `allowHalfOpen` parameter set to `false`, the ReadableStream is being read from, and the remote connection sends a FIN packet (graceful closure) or a RST packet</li>
+</ul>
```

Comment:
Aren't these equivalent? just a style preference?

Reply:
Yes, just trying to be consistent. There was `*` `-` and `<ol>` used so I switch everything to HTML